docs/core/tools/dotnet-nuget-why.md

```diff
@@ -0,0 +1,65 @@
+---
+title: dotnet nuget why command
+description: Shows the dependency graph for a particular package for a given project or solution.
+author: advay26
+ms.date: 05/30/2024
+---
+# dotnet nuget why
+
+**This article applies to:** ✔️ .NET 8.0.4xx SDK and later versions
+
+## Name
+
+`dotnet nuget why` - Shows the dependency graph for a particular package.
+
+## Synopsis
+
+```dotnetcli
+dotnet nuget why <PROJECT|SOLUTION> <PACKAGE> [-f|--framework <FRAMEWORK>]
+
+dotnet nuget why -h|--help
+```
+
+## Description
+
+The `dotnet nuget why` command shows the dependency graph for a particular package for a given project or solution.
+
+## Arguments
+
+- **`PROJECT|SOLUTION`**
+
+  A path to a project file, solution file, or directory. If a directory is specified, the command searches the directory for a project or solution file. If more than one project or solution is found, an error is thrown.
```

Comment:
Looking at https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-build, it uses the description:

> The project or solution file to build. If a project or solution file isn't specified, MSBuild searches the current working directory for a file that has a file extension that ends in either proj or sln and uses that file.

NuGet commands (should) have the same semantics, so perhaps it's worthwhile copy/pasting existing docs, rather than writing something unique.

Reply:
I tried to mimic what `dotnet list package` says: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-list-package#arguments

> The project or solution file to operate on. If not specified, the command searches the current directory for one. If more than one solution or project is found, an error is thrown.

'why' doesn't currently have the 'default-to-current-directory' behavior, so I had to adapt it.

Reply:
I can change "A path to a project file, solution file, or directory." to "The project or solution file to operate on" to be consistent